misk/src/main/kotlin/misk/security/authz/TrogdorAuthenticator.kt

```diff
@@ -0,0 +1,26 @@
+package misk.security.authz
+
+import misk.MiskCaller
+import misk.scope.ActionScoped
+import misk.web.Request
+import javax.inject.Inject
+
+/** Maps the incoming request to a human user based on headers forwarded by trogdor */
```

Comment:
should we start putting sq specific stuff in a separate package?

Reply:
I think ideally a separate repo, but that might be quite a pain with the state of our gradle build. related to the bug you filed recently about splitting everything up. so maybe just put in a separate package for now with the intention of moving out